c/factorial.c

```diff
@@ -0,0 +1,56 @@
+// You can calculate the factorial of a number in C using a recursive or iterative approach. Here's an example of both methods:
+
+//1. Using Recursion:
```

Comment:
It would be better if you make a separate file for iteration and recursion program 
Just rename the file for iteration as it is and 
Rename recursion as "factorialRecursion.c"